.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Before submitting
+
+<!--
+Before submitting a PR, please make sure to check our contributing guidelines regarding [code formatting](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#code-formatting-and-typing), [tests](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#unit-tests), and [documentation](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#documentation).
+If we didn't discuss your PR in Github issues there's a high chance it will not be merged.
+-->
+
+## What does this PR do?
+
+<!-- Please describe it here -->
```

Comment:
This will not be rendered as markdown so we can't add that many links


```suggestion
<!-- Before submitting a PR, please make sure to check our contributing guidelines regarding code formatting, tests, and documentation: https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md -->
```